models/FESOM/README.rst

```diff
@@ -0,0 +1,390 @@
+   experiment. This file is sourced by every other script below.
+
+2. *experiment.launch* Takes the information from **environment.load** and
+   creates runnable scripts from the template script files. This also initiates
+   the first cycle of the experiment.
+
+   2.1. *ensemble.sh*
+
+   2.1.1. *initialize.template* (first cycle only)
+
+   2.1.2. *advance_model.template* (job array to advance the ensemble)
+
+   2.1.3. *check_ensemble.sh* (if all goes well, assimilate)
+
+   2.1.3.1. *filter.template* (assimilate)
+
+   2.1.3.2. *finalize.sh* if all goes well and experiment is not finished ...
+            continue to 2.1
+
+Shell Scripts
+-------------
+
+Shell scripts are written in bash for LSF queuing system. They should be
+modified to work with others such as SLURM. FESOM executables are called
+externally detached from DART therefore no need for an advance model.
+
++---------------------------------------+---------+---------------------+
+| Script                                | Queue   | Definition          |
++=======================================+=========+=====================+
+| **environment.load**                  | serial  | Includes            |
+|                                       |         | environment         |
+|                                       |         | variables,          |
+|                                       |         | relevant            |
+|                                       |         | directories,        |
+|                                       |         | experiment          |
+|                                       |         | specifications.     |
+|                                       |         | This file is        |
+|                                       |         | sourced by every    |
+|                                       |         | other script        |
+|                                       |         | below.              |
++---------------------------------------+---------+---------------------+
+| **experiment.launch**                 | serial  | Main script which   |
+|                                       |         | modifies            |
+|                                       |         | ``ensemble.sh``     |
+|                                       |         | and calls           |
+|                                       |         | ``ensembl           |
+|                                       |         | e.${EXPINFO}.sh``.  |
+|                                       |         | An                  |
+|                                       |         | e                   |
+|                                       |         | xperiment-specific  |
+|                                       |         | summary which       |
+|                                       |         | should be modified  |
+|                                       |         | before launching    |
+|                                       |         | the scripts.        |
```

Comment:
The last column here (all rows) have unfortunate breaks in the words.

Reply:
Fixed.